src/testcafe-runner.ts

```diff
@@ -280,7 +282,12 @@ async function run (nodeBin: string, runCfgPath: string, suiteName: string) {
   process.env.SAUCE_SUITE_NAME = suiteName;
   process.env.SAUCE_ARTIFACTS_DIRECTORY = cfg.assetsPath;
 
-  const tcCommandLine = buildCommandLine(cfg.suite as Suite, cfg.projectPath, cfg.assetsPath);
+  // Copy our runner's TestCafe configuration to __project__/ to preserve the customer's
+  // configuration, which will be needed for future use.
```

Comment:
This doc is sort of ambiguous. "future use" would mean that we have plans for it in some kind of near or far future. But not right now. Is that what you are trying to say?

Reply:
No. Let me revise it.